app/controllers/admin/api/applications_controller.rb

```diff
@@ -24,25 +24,19 @@ def application_filter_params
   end
 
   def applications
-    @applications ||= begin
-      cinstances = current_account.provided_cinstances.where(service: accessible_services)
-      if (service_id = params[:service_id])
-        cinstances = cinstances.where(service_id: service_id)
-      end
-      cinstances
-    end
+    @applications ||= current_account.provided_cinstances.merge(accessible_services)
   end
 
   def application
     @application ||= case
 
-                     when user_key = params[:user_key]
+                     when param_key = params[:user_key]
```

Comment:
why rename `user_key` to `param_key` ?

Reply:
because underneath there is a baby_squeel expression `(user_key == param_key)`. I thought that even if it worked, it would be highly confusing to read `(user_key == user_key)`